platform/process/tasks/ABProcessTaskServiceGetResetPasswordUrl.js

```diff
@@ -0,0 +1,67 @@
+// import ABApplication from "./ABApplication"
+// const ABApplication = require("./ABApplication"); // NOTE: change to require()
+const ABProcessTaskServiceGetResetPasswordUrlCore = require("../../../core/process/tasks/ABProcessTaskServiceGetResetPasswordUrlCore.js");
+
+module.exports = class ABProcessTaskServiceGetResetPasswordUrl extends (
+   ABProcessTaskServiceGetResetPasswordUrlCore
+) {
+   /**
+    * @method do()
+    * this method actually performs the action for this task.
+    * @param {obj} instance
+    *        the instance data of the running process
+    * @param {Knex.Transaction?} trx
+    *        (optional) Knex Transaction instance.
+    * @param {ABUtil.reqService} req
+    *        an instance of the current request object for performing tenant
+    *        based operations.
+    * @return {Promise}
+    *        resolve(true/false) : true if the task is completed.
+    *                            false if task is still waiting
+    */
+   do(instance /*, trx, req */) {
+      this.stateCompleted(instance);
+
+      return new Promise((resolve, reject) => {
+         try {
```

Comment:
I don't sure how `this.AB.req.serviceRequest` throw error.

Look like it pass the error within callback function.